### ISSUE TITLE
DPL-nnn Biosero - create new views and stored procedures for the GBG summary

### ISSUE THREAD

**Issue by andrewsparkes** [sanger/cherrytrack#15]
**User story**
As Biosero we would like a stored procedure we can call that returns the summary information for a run. For use in creating the summary display panel for the users at the end of a run.

**Who are the primary contacts for this story**
Rob H
Andrew
Eduardo

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] stored procedure is created getRunSummaryInformation
- [ ] returns list of controls and source plates used in the run with counts of wells picked (in this run)
- [ ] for each source plate the count of empty wells remaining (zero for fully picked, > 0 for partially picked)
- [ ] for each source plate whether it was partially picked in any previous run(s) (a boolean) i.e. samples picked into other destination plate(s)

**Dependencies**
This story is blocked by the following dependencies:
None

References
This story has a non-blocking relationship with:
NA

**Additional context**
Difficult to achieve this in one query for a view.

**Comment by andrewsparkes** [comment]
Biosero (Rob H) have now said they will attempt to build this summary information up as they progress a run. So for now putting story on hold. They make ask for specific stored procedures to pull information that cannot be tracked in that way (e.g. if a source was used in other runs)
